0009-non-helium-hotspots.md

```diff
@@ -0,0 +1,248 @@
+- A hotspot at Level 1 is allowed to transmit data and earn HNT based only on transmitting network data.
+
+### Level 2: Asserting location on the network
+
+- A hotspot must satisfy the below minimum criteria to enter Level 2:
+    * It must be a Level 1 hotspot for X number of blocks.
+    * Issue an `assert_location` transaction by incurring a $40 USD data credit burn fee.
+    * Witness sufficient number of PoC challenges involving Level 3 and above hotspots.
+- A hotspot at Level 2 gains access to participate in PoC challenges (become a member in the PoC path).
+
+### Level 3: Gaining PoC challenge priviledges
+
+Level 3 is divided into two broad categories:
+
+#### Level 3A: Organic network growth
+
+- A hotspot must satisfy the below minimum criteria to enter Level 3A:
+    * It must be a Level 2 hotspot for X number of blocks.
+    * Successfully participate in X number of PoC challenges involving other Level 3 and above hotspots.
+    * A successful challenge comprises of the hotspot being able to receive a PoC packet from a Level 3 hotspot,
+      successfully decrypt it and successfully transmit it to a Level 2 or above hotspot.
+- A hotspot at Level 3A gains access to construct PoC challenges.
+
+#### Level 3B: On-chain governance
+
+- A hotspot must satisfy the below minimum criteria to enter Level 3B:
+    * It must be a Level 1 hotspot for X number of blocks.
+    * It must be voted in via the on-chain governance mechanism (TBD) wherein either Helium (or an authorized 3rd party)
+      countersigns the `assert_location` transaction on the hotspot owner's behalf.
+- A hotspot at Level 3B gains access to construct PoC challenges.
+
+### Level 4: Consensus candidacy
+
+- A hotspot must satisfy the below minimum criteria to enter Level 4:
+    * It must be a Level 3 hotspot for X number of blocks.
+    * The hotspot owner or another miner in the network must stake 10000 HNT in order to claim Level 4 status for the
+      hotspot.
+- A hotspot at Level 4 gains access to consensus membership candidacy.
+
+## Level Demotion
+
+With the exception of Level 3B and Level 4 hotspots, any hotspot belonging to other levels is susceptible to level
+demotion governed by the hotspot's score. If the score for a hotspot drops by X for Y blocks, it will be demoted down to
+one previous level.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+TBD
+
+# Rationale and Alternatives
+[alternatives]: #rationale-and-alternatives
+
+TBD
+
+# Unresolved Questions
+[unresolved]: #unresolved-questions
+
+1. Under what conditions do we remove the stake for Level 4 hotspots?
+2. Where does the removed stake go to? Does it just vanish?
+3. How do we actually remove the stake?
+
+# Deployment Impact
+[deployment-impact]: #deployment-impact
+
+We need to ensure we transition the existing hotspots on the network in a meaningful way. Whether every single existing
+hotspot goes into the same level or not is yet to be determined.
```

Comment:
There needs to be some number of Hotspots at L3 so challenges can be created, the rest should be L2? We need to decide the criteria to bless existing Hotspots as L3 in the new world.